gdx-backend-jtransc/src/com/badlogic/gdx/backends/lwjgl/LwjglApplicationConfiguration.java

```diff
@@ -66,8 +67,16 @@ public void setFromDisplayMode(Graphics.DisplayMode mode) {
 		this.fullscreen = true;
 	}
 
+	@HaxeMethodBody("" +
+		"{% if extra.fps %} return {{ extra.fps }}; {% end %}" +
+		"{% if !extra.fps %} return false; {% end %}"
```

Comment:
This should be 60 I think. `false` looks like a bug since the function returns an integer.
```
{% if extra.fps %} return {{ extra.fps }}; {% else %} return 60; {% end %}
```